skins/laika/src/components/shared/Email.vue

```diff
@@ -1,18 +1,19 @@
 <template>
-    <fieldset>
+    <fieldset class="has-margin-top-18">
 		<label for="email" class="subtitle">{{ $t( 'donation_form_email_label' ) }}</label>
 		<b-input type="text"
 			id="email"
 			:placeholder="$t( 'donation_form_email_placeholder' )"
 			v-model="emailValue"
 			@blur="validateEmail">
 		</b-input>
-		<span v-if="emailHasError" class="help is-danger">{{ $t( 'donation_form_email_error' ) }}</span>
+		<span v-if="showError.email || emailHasError" class="help is-danger">{{ $t( 'donation_form_email_error' ) }}</span>
```

Comment:
Hmm, this "I validate the email myself, but my validation can be overwritten from the outside" behavior looks strange to me. Do we need an email field that does standalone validation?

Reply:
Yeah, I thought the same but I had issues implementing it properly. I will give it another go.